index.html

```diff
@@ -75,112 +89,167 @@ <h2>Multi-Post-Stories</h2>
+                                  <li>Bootstrap</li>
+                                  <li>Ruby</li>
+                              </ul>
+                          </div>
+                          <div class="see-btn">
+                              <button>See Projects</button>
+                          </div>
+                    </div>
+                    
+                  </div>
+                  <div class="project-section third desk-btn">
+                    <div class="project-section-box">
+                        <h2>Website Portfolio</h2>
+                        <p>
+                            A daily selection of privately  <br> 
+                            personalized reads; no accounts or <br> 
+                            sign-ups required. Has been the <br> 
+                            industry's standard.
+                        </p>
+                        <div class="btn">
+                            <ul>
+                                <li>HTML</li>
+                                <li>Bootstrap</li>
+                                <li>Ruby</li>
+                            </ul>
+                        </div>
+                        <div class="see-btn">
+                            <button>See Projects</button>
+                        </div>
+                    </div>
+                      
+                  </div>
+              </div>
         </div>
     </section>
     <section class="about-me-section">
-        <div class="about-me">
-            <h1>About me</h3>
-            <p>
-                Hello I’m a software developer! I can help <br>
-                you build a product , feature or website <br>
-                Look through some of my work and <br> 
-                experience! If you like what you see and 
-                <br> have a project you need coded, don’t 
-                <br> hesitate to contact me.
-            </p>
-            <div class="resume-btn">
-                <button>Get my resume</button>
+            <div class="about-me">
+                <h1>About me</h3>
+                <p>
+                    Hello I’m a software developer! I can help <br>
+                    you build a product , feature or website <br>
+                    Look through some of my work and <br> 
+                    experience! If you like what you see and 
+                    <br> have a project you need coded, don’t 
+                    <br> hesitate to contact me.
+                </p>
+                <div class="resume-btn">
+                    <button>Get my resume</button>
+                </div>
             </div>
-        </div>
-        <hr>
-        <div class="about_footer">
+           <hr>
+         <div class="about_footer">
             <div class="profile">
-               <ul>
+               <ul class="ul">
                    <li class="list-style Language">Language
                        <ul>
                             <li>JavaScript</li>
```

Comment:
- [ ] Kindly look into your style for all the symbols and how they're styled on the figma design and ensure you produce a close replica of each of them on your design.

### Your Design
![your-lang](https://github.com/Blessman-Newton/Microverse_Portfolio/assets/26597432/dcbf7030-d0c4-402e-8712-15dc9fabc7d9)

### Figma Design
![temp-lang](https://github.com/Blessman-Newton/Microverse_Portfolio/assets/26597432/052a2600-d5b6-4ed4-b9a7-90facd480986)